src/components/LoginForm/index.js

```diff
@@ -72,44 +42,113 @@ function Login({ providers: providersProp, ...props }) {
     return null;
   }
   return (
-    <Grid
-      container
-      justifyContent="space-around"
-      alignItems="center"
-      className={classes.root}
-    >
-      <Grid item xs={12} className={classes.item}>
-        <form noValidate className={classes.formStyles}>
+    <Section className={classes.section}>
+      <Grid container>
+        <Grid item xs={12} md={7} className={classes.container}>
+          <Typography variant="h2">{title}</Typography>
+          <Typography className={classes.text}>{description}</Typography>
+          <form className={classes.form}>
+            <TextField
+              className={classes.textfield}
+              InputLabelProps={{ className: classes.label }}
+              InputProps={{ className: classes.input }}
+              autoComplete="email"
+              name="email"
+              variant="outlined"
+              fullWidth
+              id="email"
+              label="Email"
+              autoFocus
+            />
+
+            <TextField
+              className={classes.textfield}
+              InputLabelProps={{ className: classes.label }}
```

Comment:
```suggestion
              InputLabelProps={{ className: classes.label, shrink:false }}
```
To disable the shrink on focus